docs/manual.rst

```diff
@@ -2301,7 +2301,31 @@ filters.typogrify_sans_widont
    Same as typogrify without the widont filter
 
 filters.typogrify_custom
-    Run typogrify with a custom set or filters. Takes ``typogrify_filters`` (a list of callables) and ``ignore_tags`` (defaults to None).
+    Run typogrify with a custom set of filters or ignored tags. Takes ``typogrify_filters``
+    (a list of callables) or ``ignore_tags`` (a list of HTML tags to ignore). These must be
+    specified as keyword arguments to ``functools.partial``. The following code should be
+    placed in ``conf.py``.
+
+    .. code-block:: python
+
+      from nikola.filters import typogrify_custom
+      from functools import partial as ft_partial
```

Comment:
Do we really need an `as` import here? Much easier to just `import functools`.

Reply:
I did this to reduce the line length of the example. Happy to change it back.

Reply:
Clarity is much more important than line length. I personally recommend avoiding `as` except for a few established aliases (`import numpy as np; import pandas as pd`) and avoiding naming conflicts (as we do with `typogrify`in `filters.py`).